### DISCUSSION TITLE
Is there a way to specify the path of `.cache` directory?

### DISCUSSION THREAD

**Discussion by QuarticCat** [clangd/clangd#1187] [Q&A] [answered]
I prefer collecting cache in a folder so that I can use systemd-tmpfiles to automatically clean the old cache. ccls has this feature [MaskRay/ccls/wiki/Visual-Studio-Code#cclscachedirectory](https://github.com/MaskRay/ccls/wiki/Visual-Studio-Code#cclscachedirectory)

**Comment by HighCommander4** [comment, marked as answer]
There is not currently, it's placed next to the `compile_commands.json`.

#184 tracks customizing this location, and its marked as a "good first issue", so patches are welcome.
